guillotina/traversal.py

```diff
@@ -447,9 +447,13 @@ async def real_resolve(self, request: IRequest) -> Optional[MatchInfo]:
                     )
 
         if not view and len(tail) > 0:
-            # we should have a view in this case because we are matching routes
-            await notify(TraversalViewMissEvent(request, tail))
-            raise HTTPNotFound(content={"reason": "object and/or route not found"})
+            # Try arbitrary "path" in the path
+            view_name = tail[0] + "?"
+            view = query_multi_adapter((resource, request), method, name=view_name)
+            if not view:
```

Comment:
Sorry, another tweak. Can you change `not view` to `view is None`?

Also, can you fix the `not view` on line 449?

Reply:
ah sure!